scripts/circleci/verify-circle-ci-config.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
I guess this file should be named verify-codecov-config.sh?

Reply:
Yep, thanks for catching this :)